pihole

```diff
@@ -481,41 +482,41 @@ Options:
   echo -e "${OVER}  ${TICK} ${str}"
 }
 
-piholeStatus() {
-  if [[ "$(netstat -plnt | grep -c ':53 ')" -gt "0" ]]; then
-    if [[ "${1}" != "web" ]]; then
-      echo -e "  ${TICK} DNS service is running"
-    fi
+statusFunc() {
+  local addnConfigs
+
+  # Determine if service is running on port 53
+  if nc -z 127.0.0.1 53; then
+    [[ "${1}" != "web" ]] && echo -e "  ${TICK} DNS service is running"
```

Comment:
Just for style this would be better in an `if [[ ]]; then; else` in either a multi line statement or if you want to keep it oneliner then in that format.